src/main/groovy/org/cristalise/dsl/collection/DependencyDelegate.groovy

```diff
@@ -52,7 +52,7 @@ class DependencyDelegate {
     public void Member(Map attrs, Closure cl = null) {
         assert attrs && attrs.itemPath
 
-        def member = dependency.addMember(new ItemPath((String)attrs.itemPath))
+        def member = dependency.addMember(new ItemPath((String)attrs.itemPath, true))
```

Comment:
this should be using DomainPath instead of ItemPath

Reply:
But, Dependency#addMember accepts only ItemPath as the constructor argument.